src/config/backpack/base.php

```diff
@@ -56,6 +56,7 @@
     */
 
     // The prefix used in all base routes (the 'admin' in admin/dashboard)
+    // You make sure all your URLs use this prefix by using the backpack_url() helper instead of url()
```

Comment:
There seems to be a `can` missing after `You`?

Reply:
Or just not-needed `You` :)